### ISSUE TITLE
1.1.0 Crash on get_player

### ISSUE THREAD

**Issue by Heuvend** [JDOGG88/Logistic-Machines#2]
The mod Logistic Machines (1.1.0) caused a non-recoverable error.
Please report this error to the mod author.
```
Error while running event Logistic-Machines::on_robot_built_entity (ID 14)
__Logistic-Machines__/control.lua:349: bad argument #2 of 2 to 'get_player' (string expected, got nil)
stack traceback:
	[C]: in function 'get_player'
	__Logistic-Machines__/control.lua:349: in function 'build_entity'
	__Logistic-Machines__/control.lua:358: in function <__Logistic-Machines__/control.lua:357>
```

When did this happen:
First build of construction robots (Pipe / Belt)

Other:
- Krastorio 2 mod was enabled
- Reloading autosave, placing a single belt in the logistic buffer, and wait for the bot to build triggered the crash again

Full mod list:
```
{
  "mods": 
  [
    
    {
      "name": "base",
      "enabled": true
    },
    
    {
      "name": ,
      "enabled": true
    },
    
    {
      "name": "accumulator-wagon",
      "enabled": true
    },
    
    {
      "name": "AutoDeconstruct",
      "enabled": true
    },
    
    {
      "name": "BeastFinder",
      "enabled": true
    },
    
    {
      "name": "car-finder",
      "enabled": true
    },
    
    {
      "name": "Command_and_Conquer_RedAlert_Sounds",
      "enabled": true
    },
    
    {
      "name": "concentrated-solar-power",
      "enabled": true
    },
    
    {
      "name": "CorpseFlare",
      "enabled": true
    },
    
    {
      "name": "EpicArtillerySounds",
      "enabled": true
    },
    
    {
      "name": "fireproof-bots",
      "enabled": true
    },
    
    {
      "name": "flib",
      "enabled": true
    },
    
    {
      "name": "FluidMustFlow",
      "enabled": true
    },
    
    {
      "name": "InfiniteInventory",
      "enabled": true
    },
    
    {
      "name": "inventory-repair",
      "enabled": true
    },
    
    {
      "name": "K2_HMG_Turret",
      "enabled": true
    },
    
    {
      "name": "Krastorio2",
      "enabled": true
    },
    
    {
      "name": "LaneBalancer",
      "enabled": true
    },
    
    {
      "name": "Logistic-Gun-Turret",
      "enabled": true
    },
    
    {
      "name": "Logistic-Machines",
      "enabled": true
    },
    
    {
      "name": "quarry-18",
      "enabled": true
    },
    
    {
      "name": "RateCalculator",
      "enabled": true
    },
    
    {
      "name": "RecipeBook",
      "enabled": true
    },
    
    {
      "name": "SchallMachineScaling",
      "enabled": true
    },
    
    {
      "name": "SchallPipeScaling",
      "enabled": true
    },
    
    {
      "name": "SchallRecipeScaling",
      "enabled": true
    },
    
    {
      "name": "shield-generators",
      "enabled": true
    },
    
    {
      "name": "Shortcuts-ick",
      "enabled": true
    },
    
    {
      "name": "very-long-handed-inserter",
      "enabled": true
    },
    
    {
      "name": "WipMod_HeavyTank",
      "enabled": true
    }
  ]
}
```

**Comment by JDOGG88** [comment]
The function needs to be fixed, will work on it but in the mean time just don't use robots to place machines.  I know that's a shitty answer but that's what's causing the error.

**Comment by JDOGG88** [comment]
Fixed in update v1.1.1
